pages/content/amp-dev/documentation/guides-and-tutorials/contribute/contribute-documentation/terminology.md

```diff
@@ -0,0 +1,149 @@
+---
+$title: AMP terminology 
+$order: 2
+description: 'AMP terminology guidelines'
+formats:
+  - websites
+  - stories
+  - ads
+  - email
+author: CrystalOnScript
+---
+
+<table>
+  <tr>
+   <td><strong>Term</strong>
+   </td>
+   <td><strong>Definition</strong>
+   </td>
+   <td><strong>Example </strong>
+   </td>
+  </tr>
+  <tr>
+   <td>AMP
+   </td>
+   <td>Official name of the open-source web component framework.
+<p>
+Accelerated Mobile Pages was the expansion of AMP. The deprecated term is too narrow of a definition for AMP:
+<p>
+AMP doesn’t just power pages anymore – you can now build ads, emails and stories with it.
+   </td>
+   <td><ul>
+
+<li>AMP controls all resource downloads.
+<li>Twitter supports AMP.</li></ul>
+
+   </td>
+  </tr>
+  <tr>
+   <td>AMP Project
+   </td>
+   <td>Refers to the open-source initiative to make the web better for all by building a performance-focused ecosystem comprising of publishers and technology providers.
+<p>
+Use “AMP Project” when talking about the ecosystem or collaborative effort across multiple partners.  Do not use "AMP Project" to refer to the actual technology, instead use AMP.
+<p>
+Note: "Project" is capitalized.
+   </td>
+   <td><ul>
+
+<li>The AMP Project is a collaborative and open source effort.
+<li>By participating in the AMP Project, consumption platforms ensure that the open web thrives.</li></ul>
+
+   </td>
+  </tr>
+  <tr>
+   <td>AMP website(s)
+   </td>
+   <td>A website, or websites, built using the AMP framework. 
+   </td>
+   <td><ul>
+
+<li>AMP websites are fast, dynamic, and put user’s first.
+<li>Building an AMP website is easy and intuitive. </li></ul>
+
+   </td>
+  </tr>
+  <tr>
+   <td>AMP components
+   </td>
+   <td>Refers to a particular feature (“web component”) that works only* when used on an AMP-powered page, and comes with their own HTML notation.
+<p>
+Examples include: amp-form, amp-img, amp-video, ect.
+   </td>
+   <td><ul>
+
+<li>The built-in AMP components, such as amp-img, amp-video, and amp-pixel are included in the base library.
+<li>The amp-analytics component can be used to measure activity on an AMP document.</li></ul>
+
+   </td>
+  </tr>
+  <tr>
+   <td>AMP cache
+   </td>
+   <td>A cache that support AMP. 
+   </td>
+   <td><ul>
+
+<li>When an AMP cache serves your page, it isn’t loaded from your site’s origin. 
+<li>Google and Cloudflare both have AMP caches. </li></ul>
+
+   </td>
+  </tr>
+  <tr>
+   <td>AMP story / AMP stories 
+   </td>
+   <td>A story web experience built with AMP. 
+   </td>
+   <td><ul>
+
+<li>An AMP story format is free and part of the open web.
+<li>AMP stories immerse readers in fast-loading full-screen experiences.  </li></ul>
+
+   </td>
+  </tr>
+  <tr>
+   <td>AMP ads
+   </td>
+   <td>An ad built using the AMPHTML ad spec. 
+   </td>
+   <td><ul>
+
+<li>AMP ads greatly improve the mobile internet through their intelligent design and pre-loading features.</li></ul>
```

Comment:
mobile internet -> mobile web (I don't think they're shown much outside of the web)